langstream-runtime/langstream-runtime-impl/src/main/python/langstream_grpc/__main__.py

```diff
@@ -27,12 +27,15 @@
         datefmt="%H:%M:%S",
     )
 
-    if len(sys.argv) != 3:
+    if len(sys.argv) <= 3:
         print("Missing gRPC target and python class name")
         print("usage: python -m langstream_grpc <target> <config>")
         sys.exit(1)
 
-    server = AgentServer(sys.argv[1], sys.argv[2])
+    context_config = {}
+    if len(sys.argv) > 3:
```

Comment:
the process is always launched by the same version of the java runtime, there is no need to do this

Reply:
👍